src/chart/boxplot/BoxplotView.ts

```diff
@@ -180,11 +184,19 @@ function updateNormalBoxData(
 
     el.useStyle(data.getItemVisual(dataIndex, 'style'));
     el.style.strokeNoScale = true;
-
     el.z2 = 100;
 
     const itemModel = data.getItemModel<BoxplotDataItemOption>(dataIndex);
     const emphasisModel = itemModel.getModel('emphasis');
+    const labelModel = seriesModel.get('label');
+
+    if (labelModel.show) {
+        const formattedLabels =
+            ((seriesModel as BoxplotSeriesModel).getRawValue(dataIndex) as number[])
+            .splice(1).map((value: number) => labelModel.formatter(value).toString());
```

Comment:
This is not the correct way to format labels. If `formatter` is not defined, this should go wrong. You should probably do like [sunburst](https://github.com/apache/echarts/blob/master/src/chart/sunburst/SunburstPiece.ts#L189).

Reply:
@Ovilia I am not sure if I understood correctly - `getFormattedLabel` is defined to return a `string` but somehow returns an `object` when being called with `dataIndex`.

I think I am missing something here, could you clarify how you would implement the formatting of the labels?

Thanks